packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -268,6 +270,16 @@ module.exports = {
     // having to parse `index.html`.
     new ManifestPlugin({
       fileName: 'asset-manifest.json'
+    }),
+    // Generate and inject subresources hashes in the final `index.html`.
+    new SubresourceIntegrityPlugin({
```

Comment:
Just a reminder before merging: this was temporarily disabled in https://github.com/facebookincubator/create-react-app/pull/1233/files